samples/bluetooth/mesh/light_dimmer/prj.conf

```diff
@@ -49,10 +48,9 @@ CONFIG_BT_MESH_GATT_PROXY=y
 CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME=y
 CONFIG_BT_MESH_DK_PROV=y
 CONFIG_BT_MESH_NLC_PERF_CONF=y
-# Disable model extension API and set list size to 0 as the models in this sample don't have any
-# relation between each other.
+# Disable model extension API as the models in this sample don't have any relation between
+# each other.
```

Comment:
`CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE` defaults to 10. I think `CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE=0` should stay.

Reply:
It defaults to 0 if `!BT_MESH_MODEL_EXTENSIONS`?